project/authentication/forms.py

```diff
@@ -12,18 +12,18 @@
 from django.utils.translation import ugettext_lazy as _
 
 from api.tasks import send_email as task_send_email
-from .reserved_usernames import RESERVED_USERNAMES
+from reserved_usernames import RESERVED_USERNAMES
```

Comment:
We recently had a [recent pull request](https://github.com/CiviWiki/OpenCiviWiki/pull/74) where we converted imports to use relative import (".") semantics. This was to align with the 'best practice' recommendation in the book [Two Scoops of Django](https://twoscoopspress.com/products/two-scoops-of-django-1-11).

Make sure this change is desired.

Reply:
At this time, yes, this change is desired.  It reverts the Python 2 to 3 changes in the 2to3 branch, which isn't yet committed.

The reversions can be recommitted, if desired.

Reply:
OK, cool beans. Just checking :-) Revert the changes whenever is convenient.